ganga/GangaDirac/test/Dockerfile

```diff
@@ -16,20 +16,20 @@ RUN virtualenv venv && \
 
 RUN mkdir dirac_ui &&\
     cd dirac_ui &&\
-    wget -np -O dirac-install https://raw.githubusercontent.com/DIRACGrid/DIRAC/rel-v6r19/Core/scripts/dirac-install.py &&\
+    wget -np -O dirac-install https://raw.githubusercontent.com/DIRACGrid/DIRAC/integration/Core/scripts/dirac-install.py &&\
```

Comment:
This doesn't work due to:
https://github.com/DIRACGrid/DIRAC/issues/3906

Reply:
Just fixed that

Reply:
test please